backend/pick-git/src/main/java/com/woowacourse/pickgit/post/infrastructure/extractor/GithubRepositoryExtractor.java

```diff
@@ -37,7 +39,7 @@ public GithubRepositoryExtractor(
 
     private String generateApiUrl(String username, Pageable pageable) {
         return String
-            .format(API_URL_FORMAT, username, pageable.getPageNumber() + 1, pageable.getPageSize());
+            .format(apiUrlFormatForRepository, username, pageable.getPageNumber() + 1, pageable.getPageSize());
```

Comment:
```suggestion
            .format(
               apiUrlFormatForRepository, 
               username,  
               pageable.getPageNumber() + 1, 
               pageable.getPageSize()
            );
```

다른 클래스에는 이렇게 되어있길래 사소한거지만 남겨요 !